package.json

```diff
@@ -65,6 +66,7 @@
     "@types/node": "^22.15.3",
     "@types/react": "19.1.2",
     "@types/react-dom": "19.1.3",
+    "@types/uuid": "^10.0.0",
```

Comment:
This library was used in the 786 branch but it has now been removed. Try pulling the changes from branch 786 again, so that you have the current version and don't add unnecessary dependencies